worker/src/durable.rs

```diff
@@ -176,6 +176,15 @@ impl ObjectId<'_> {
             })
             .map_err(Error::from)
     }
+
+    /// The name that was used to create the `ObjectId` via [`id_from_name`](https://developers.cloudflare.com/durable-objects/api/namespace/#idfromname).
```

Comment:
Should we also implement `equals(otherId)` as well here?

Reply:
Done! The equality check ignores the optional namespace field of the ObjectId, but still returns false when comparing two ObjectIds with the same name but from different namespaces, which is the behavior I'd expect.